robottelo/vm.py

```diff
@@ -293,6 +297,17 @@ def put(self, local_path, remote_path=None):
             )
         ssh.upload_file(local_path, remote_path, hostname=self.ip_addr)
 
+    def fetch_domain(self):
```

Comment:
This method is not related to the VM, we can rename it for now to provisioning_server_hostname.

The next step should be receive the domain as configuration and if not provided fetch it from the provisioning server..

Also the provisining server domain is the same as:

```python
domain = self.provisioning_server.split('.', 1)[1]
```